README.md

```diff
@@ -96,7 +96,17 @@ SparsePie_tuple(rhubarb=None, cherry=None, mud=None)
 SparsePie_tuple(rhubarb=2, cherry=1, mud=None)
 ```
 
-Use `SparseEnumap.set_defaults` to set more useful default values:
+Use `enum.default()` to declaratively specify defaults for missing values:
```

Comment:
This should be `enumap.default()`, right?

Reply:
Indeed. Thank you.